sql/parser/sql.y

```diff
@@ -2807,6 +2808,14 @@ func_expr_common_subexpr:
 | GREATEST '(' expr_list ')' { unimplemented() }
 | LEAST '(' expr_list ')' { unimplemented() }
 
+func_or_variable:
```

Comment:
Only current_date and current_time have the func_or_variable treatment in postgres; the others are only functions. I'd prefer to minimize the number of names that have this special case.